### PR TITLE
Source code refining and cleaning

### PR DESCRIPTION
**Pull request by dkozinski**
Removed useless source code sections
Improved comments lines

### CODE REVIEW

File: ffmpeg-subtree/libavcodec/libxeve.c [hunk 1]
```diff
@@ -162,6 +162,49 @@ static int get_tune_id(const char *tune)
...
+        break;
+    case AV_PIX_FMT_YUV444P10:
+        *color_format = XEVE_CF_YCBCR444;
+        *bit_depth = 10;
+        break;
+    default:
+        *color_format = XEVE_CF_UNKNOWN;
+        return -1;
+    }
+    return 0;
```
Comment (dariusz-f) [inline ffmpeg-subtree/libavcodec/libxeve.c:206]: add empty line before return

File: ffmpeg-subtree/libavcodec/libxeve.c [hunk 2]
```diff
@@ -249,155 +296,108 @@ static int parse_xeve_params(const char *key, const char *value, XeveContext *xe
...
-        break;
-    case AV_PIX_FMT_YUV444P10:
-        *color_format = XEVE_CF_YCBCR444;
-        *bit_depth = 10;
-        break;
-    default:
-        *color_format = XEVE_CF_UNKNOWN;
-        return -1;
-    }
     return 0;
```
Comment (dariusz-f) [inline ffmpeg-subtree/libavcodec/libxeve.c:317]: add empty line before return

File: ffmpeg-subtree/libavcodec/libxeve.c [hunk 3]
```diff
@@ -805,6 +806,12 @@ static int xeve_color_space(enum AVPixelFormat pix_fmt)
     return cs;
 }
 
+/**
+ * @brief Switch encodet to bumping mode
```
Comment (dariusz-f) [inline ffmpeg-subtree/libavcodec/libxeve.c]: encodet -> encoder

File: ffmpeg-subtree/libavcodec/libxeve.c [hunk 4]
```diff
@@ -805,6 +806,12 @@ static int xeve_color_space(enum AVPixelFormat pix_fmt)
...
+ * 
+ * @param id XEVE encodec instance identifier
```
Comment (dariusz-f) [inline ffmpeg-subtree/libavcodec/libxeve.c:814]: encodec -> encoder

File: ffmpeg-subtree/libavcodec/libxeve.c [hunk 5]
```diff
@@ -897,27 +893,27 @@ static av_cold int libxeve_init(AVCodecContext *avctx)
...
-    xe->bytes_total = 0;
-    xe->state = STATE_ENCODING;
-    xe->packet_count = 0;
-    xe->bitrate = 0;
+    xectx->encod_frames = 0;
+    xectx->bytes_total = 0;
+    xectx->state = STATE_ENCODING;
+    xectx->packet_count = 0;
+    xectx->bitrate = 0;
     return 0;
```
Comment (dariusz-f) [inline ffmpeg-subtree/libavcodec/libxeve.c:922]: add empty line before return, and the same in return below

**Review by dariusz-f — CHANGES_REQUESTED** [review]

File: ffmpeg-subtree/libavcodec/libxeve.c (hunk 2 above)
Comment (dkozinski) [reply ffmpeg-subtree/libavcodec/libxeve.c:317]: Done

File: ffmpeg-subtree/libavcodec/libxeve.c (hunk 3 above)
Comment (dkozinski) [reply ffmpeg-subtree/libavcodec/libxeve.c]: Done

File: ffmpeg-subtree/libavcodec/libxeve.c (hunk 4 above)
Comment (dkozinski) [reply ffmpeg-subtree/libavcodec/libxeve.c:814]: Done

File: ffmpeg-subtree/libavcodec/libxeve.c (hunk 5 above)
Comment (dkozinski) [reply ffmpeg-subtree/libavcodec/libxeve.c:922]: Done

**Review by dariusz-f — APPROVED** [review]
good

**Pull request merged**